jax/_src/numpy/fft.py

```diff
@@ -17,16 +17,19 @@
 
 from jax import lax
 from jax.lib import xla_client
+from jax.api_util import _ensure_index_tuple
 from .util import _wraps
 from . import lax_numpy as jnp
 from jax import ops as jaxops
 
 
 def _fft_core(func_name, fft_type, a, s, axes, norm):
-  # TODO(skye): implement padding/cropping based on 's'.
   full_name = "jax.numpy.fft." + func_name
+
   if s is not None:
-    raise NotImplementedError("%s only supports s=None, got %s" % (full_name, s))
+    s = _ensure_index_tuple(s)
```

Comment:
For `fftn`, I think we need to be more careful: numpy's version raises an error if `s` is not a tuple.

Reply:
Numpy just calls `list(s)`, seems sensible here as well.